recipes/install.rb

```diff
@@ -393,17 +393,38 @@
     user "root"
     code <<-EOH
     set -e
-    if [ ! -d "/cgroup" ] ; then
-       mkdir /cgroup
+    if [ ! -d "/sys/fs/cgroup/cpu/hops-yarn" ] ; then
+       mkdir /sys/fs/cgroup/cpu/hops-yarn
     fi
-    mount -t cgroup -o cpu cpu /cgroup
+    if [ ! -d "/sys/fs/cgroup/devices/hops-yarn" ] ; then
+       mkdir /sys/fs/cgroup/devices/hops-yarn
+    fi
+    # mount -t cgroup -o cpu cpu /cgroup
     touch #{cgroups_mounted}
   EOH
      not_if { ::File.exist?("#{cgroups_mounted}") }
   end
 
 end
 
+bash 'update_owner_for_gpu' do
+  user "root"
+  code <<-EOH
+    set -e
+    chown root:#{node.hops.group} #{node.hops.dir}
```

Comment:
This might cause big problems. Loads of our other programs are under /srv/hops. If they are not members of the hops.group, they might not be able to change directory into /srv/hops. Have you tested this?